src/NodeApi/Engines/NodejsPlatform.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+using static Microsoft.JavaScript.NodeApi.JSNativeApi.Interop;
+#if !NET7_0_OR_GREATER
+using NativeLibrary = Microsoft.JavaScript.NodeApi.NativeLibrary;
+#endif
+
+namespace Microsoft.JavaScript.NodeApi.Engines;
+
+/// <summary>
+/// Manages a Node.js platform instance, provided by `libnode`.
+/// </summary>
+/// <remarks>
+/// Only one Node.js platform instance can be created per process. Once the platform isdisposed,
+/// another platform instance cannot be re-initialized. One or more <see cref="NodejsEnvironment" />
+/// instances may be created using the platform.
+/// </remarks>
+public sealed class NodejsPlatform : IDisposable
```

Comment:
>Nodejs [](http://example.com/codeflow?start=20&length=6)

I wonder if we should use use term 'NodeJS' or `Node` instead. I just did not see `nodejs` as one word used often.

Reply:
Hmm... I just checked Node.js docs and they refer everywhere to Node.js term. You are right. Please ignore my comment.